Documentation/CRDs/Object-Storage/ceph-object-store-user-crd.md

```diff
@@ -60,3 +60,88 @@ spec:
     * `user-policy`
     * `odic-provider`
     * `ratelimit`
+
+### CephObjectStoreUser Reference Secret
+
+A Kubernetes secret is created by Rook with the name, `rook-ceph-object-user-<store-name>-<user-name>` in the same namespace of cephobjectUser, where:
+
+* `store-name`: The object store name in which the user will be created. This matches the name of the objectstore CRD.
+* `user-name`: The metadata name of the cephObjectStoreUser
```

Comment:
CephObjectStoreUser is the name of a Rook-defined custom resource and should always be documented in UpperCamelCase format. lowerCamelCase should be used for the names of fields within.

```suggestion
* `user-name`: The metadata name of the CephObjectStoreUser
```